internal/configs/version1/templates_test.go

```diff
@@ -169,3 +169,47 @@ func TestMainForNGINX(t *testing.T) {
 		t.Fatalf("Failed to write template %v", err)
 	}
 }
+
+func TestSplitHelperFunctions(t *testing.T) {
+	const tpl = `{{range $n := splitinput (index .) ","}}{{$n}} {{end}}`
+
+	tmpl, err := template.New("testTemplate").Funcs(helperFunctions).Parse(tpl)
+	if err != nil {
+		t.Fatalf("Failed to parse template: %v", err)
+	}
+	var buf bytes.Buffer
+
+	slice := "foo,bar"
+	expected := "foo bar "
+	err = tmpl.Execute(&buf, slice)
+	t.Log(buf.String())
```

Comment:
Consider removing it here. 
The other tests do not check the generated config against the expected output -- they only check that there is no error returned.  That's mostly why `t.Log(buf.String())`, so that the output could be checked visually. In contrast, this test checks the output against the expected output, so no printing is necessary.